### PR TITLE
fix: adjust `LLMQ_TEST_DIP0024` params, `mine_cycle_quorum` should use correct size

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
Small dip0024 related cleanups, regtest only.

## What was done?
pls see individual commits

## How Has This Been Tested?
run tests

## Breaking Changes
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: src/llmq/params.h [hunk 1]
```diff
@@ -202,7 +202,7 @@ static constexpr std::array<LLMQParams, 14> available_llmqs = {
         .useRotation = true,
         .size = 4,
         .minSize = 4,
-        .threshold = 2,
```
[the remark this answers is not in the document]
Comment (UdjinM6) [reply src/llmq/params.h:205]: that's a regtest-only quorum

[the remark this answers is not in the document]
Comment (ogabrielides) [reply src/llmq/params.h:205]: This LLMQ type is used only on regtest. The one for devnet is: LLMQ_DEVNET_DIP0024

[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/llmq/params.h:205]: Sounds good 👍

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**